amundsen_application/static/js/components/NavBar/styles.scss

```diff
@@ -17,21 +17,41 @@
   }
 
   a {
-   display: inline-block;
-   height: 100%;
-   line-height: 32px;
-   text-decoration: none;
-   color: white;
-   padding-left: 8px;
-   padding-right: 8px;
+    display: inline-block;
+    height: 100%;
+    line-height: 32px;
+    text-decoration: none;
+    color: white;
+    padding-left: 8px;
+    padding-right: 8px;
+
+    &:hover:not(.nav-bar-avatar-link) {
+      border-bottom: 4px solid white;
+    }
 
-   &:hover {
+    &.active {
      border-bottom: 4px solid white;
-   }
+    }
 
-   &.active {
-      border-bottom: 4px solid white;
-   }
+    /* Circular Hover on Avatar */
+    &:hover.nav-bar-avatar-link {
+      /* reposition so element appears to stay in place */
+      margin-left: 4px;
+      margin-right: -4px;
+    }
+    &:hover .nav-bar-avatar {
+      /* override border-bottom style */
+      border-bottom: none;
+
+      /* circular background */
+      background-color: white;
+      border-radius: 50%;
+      padding: 4px;
+
+      /* reposition so element appears to stay in place */
+      margin-top: -4px;
+      margin-left: -8px;
```

Comment:
Is this because the circle border/padding push things out of place on hover? I would suggest always including the padding so the element never changes size. This can prevent things from shifting around on hover.

An alternative would be to use the `:after` pseudo selector and use that for the border so it's actually a separate element.

Reply:
I'll revisit and see what we can do.

Reply:
Went with alway having the padding and it did simplify things.